### ISSUE TITLE
Failed to collect dependencies at net.openhft:chronicle-map

### ISSUE THREAD

**Issue by championbarca** [dan-lind/igzplugin#5]
Hi, 

While building project using command mvn install, I am getting following errors:

I am new to java, so I can miss very obvious thing, please help me resolve.

_[ERROR] Failed to execute goal on project igplugin: Could not resolve dependencies for project com.danlind.igzplugin:igplugin:jar:0.32: Failed to collect dependencies at net.openhft:chronicle-map:jar:3.13.0: Failed to read artifact descriptor for net.openhft:chronicle-map:jar:3.13.0: Could not transfer artifact net.openhft:third-party-bom:pom:3.5.7 from/to central (http://repo.maven.apache.org/maven2): Transfer failed for http://repo.maven.apache.org/maven2/net/openhft/third-party-bom/3.5.7/third-party-bom-3.5.7.pom 501 HTTPS Required ->_ 

Real concern:
I able to connect using IG's Java sample application to my account, but unable to connect using same settings to IG via Zorro using this plugin. So trying to build this plugin with debug logs and understand what is happening. Please do suggest if there is any other best way to do it.

**Comment by championbarca** [comment]
ignored somehow able to build it. 
While login from Zorro to IG, I am getting following error:

2020-12-27 01:09:06.091 [RxComputationThreadPool-9] ERROR c.danlind.igz.adapter.RestApiAdapter - Exception when logging in, API call failed with status code 400 and body {"errorCode":"error.public-api.failure.encryption.required"}
org.springframework.web.client.HttpClientErrorException: 400 Bad Request

If you have any clue about this?

It seems IG Singapore requires V2 authentication API.

**Comment by dan-lind** [comment]
Yeah, I've has one other user from Singapore with the same issue. Unfortunately IG did not get back to me when I inquired why this didn't work with V3.

**Comment by championbarca** [comment]
thanks for reply. I will investigate because it seems IG sample client is working, so mostly I can compare and make it working.

**Comment by championbarca** [comment]
So following is difference between V2 and V3:
authRequest.setEncryptedPassword(encrypt);

V2 requires encrypted password in authentication request, that's why it fails.
I am trying to fix it, but due to refreshtoken scheduler I am stuck. *which is required for V3 but not for V2)
If you can help here, would be great.

**Comment by dan-lind** [comment]
V3 doesn't support setting encrypted passwords at all, which is the problem. Nothing to do about it really
